### PR TITLE
[ARTS-362] init service timing

### PR DESCRIPTION
**Pull request by guybartal**
## Description
- Init service should run only once and after its dependent services up & ready. (retry/wait for dependent service readiness is not - implemented yet)
- the config should be fetched from the config server and not directly from the git repo. (not implemented yet)
- init service needs an identity to call to dependent services. (not implemented yet)

### Changes
- [ARTS-362] - init service deployment changed from app service to azure container instance
- SQL password complexity changed to comply with SQL server password policy

### Checklist:
- [✔] Branch name follows convention (feature/arts-###-short-name OR fix/arts-###-short-name)
- [✔] I have included a short-meaningful title, description and changes for this PR

[ARTS-362]: https://ndph-arts.atlassian.net/browse/ARTS-362

### CODE REVIEW

**Comment by guybartal** [comment]
ignore the test trail configs guy01+guy02,
I won't include them in the final PR

File: guy01_trial/definition.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (eladiw) [inline guy01_trial/definition.json:1]: please delete this trial (not needed)

File: terraform/trial_rg/modules/fhir/main.tf [hunk 2]
```diff
@@ -2,6 +2,9 @@ resource "random_password" "fhir_sql_password" {
   length = 16
   special = true
   override_special = "_%@"
+  min_upper        = 1
```
Comment (eladiw) [inline terraform/trial_rg/modules/fhir/main.tf:5]: is this because the sporadic failure?

**Review by eladiw — COMMENTED** [review]
looks good!

File: terraform/trial_rg/modules/fhir/main.tf (hunk 2 above)
Comment (guybartal) [reply terraform/trial_rg/modules/fhir/main.tf:5]: correct, it happened to me 2-3 times so I looked at the SQL policy and matched it.

**Comment by guybartal** [comment]
We have decided to deploy this service In Azure app service because ACI doesn't yet support MSI in VNET, so I'm closing this PR.

**Pull request closed without merging**